dll/cpl/appwiz/createlink.c

```diff
@@ -597,11 +597,38 @@ ShowCreateShortcutWizard(HWND hwndCPl, LPWSTR szPath)
     return TRUE;
 }
 
+/* InitCommonControls */
+typedef void (WINAPI *FN_InitCommonControls)(void);
+
+LONG CALLBACK
+ShowCreateShortcutWizardComCtl32(HWND hwndCPl, LPCWSTR szPath)
+{
+    HINSTANCE hComCtl32;
+    FN_InitCommonControls fn;
+    LONG ret;
+
+    hComCtl32 = LoadLibraryW(L"comctl32.dll");
+#if 1
+    fn = (FN_InitCommonControls)GetProcAddress(hComCtl32, (LPSTR)(DWORD_PTR)15);
+#else
+    fn = (FN_InitCommonControls)GetProcAddress(hComCtl32, "InitCommonControls");
+#endif
+    if (fn)
+        (*fn)();
+    else
+        MessageBoxW(NULL, L"InitCommonControls failed.", L"appwiz.cpl", MB_ICONERROR);
```

Comment:
I'm quite allergic to such message boxes that give no meaningful information whatsoever about the failure. Either populate the box description with a more concise detail (and make sure you don't hard-code the box description, just implement it within resources) OR print out a debug info instead.

Reply:
Also since execution can continue even if that function failed it's useless indeed to display an interruptive msgbox.